src/applications/check-in/locales/en/translation.json

```diff
@@ -335,6 +335,6 @@
   "appointments-at_other": "appointments at",
   "we-ask-this-question-because-staff-can-call-you": "We ask this question because staff can call you back anytime after you complete check-in. If you’re more than 5-10 minutes away after completing check-in, see a staff member when you arrive.",
   "have-you-arrived-at-your-va-health-facility": "Have you arrived at your VA health facility?",
-  "the-staff-can-call-you-back": "The staff can call you back anytime now that you’ve completed check-in.",
```

Comment:
you can also remove this one from the other translations since it is orphaned now

Reply:
It apparently never got translated, I don't see it in the other translations